riff-raff.yaml

```diff
@@ -96,3 +96,16 @@ deployments:
           BuiltBy: amigo
           AmigoStage: PROD
           Recipe: grid-imgops
+
+  elasticsearch-ami-update:
+    type: ami-cloudformation-parameter
+      parameters:
+        cloudFormationStackByTags: false
+        cloudFormationStackName: media-service-elasticsearch
+        prependStackToCloudFormationStackName: false
+        amiEncrypted: true
+        amiParametersToTags:
+          ElasticSearchAMI:
```

Comment:
This matches the [stack's parameter name](https://github.com/guardian/editorial-tools-platform/blob/6fde6da0778502b621a31d19bc69189d61abd4e3/cloudformation/media-service-account/grid/elasticsearch/elasticsearch-stack.yaml#L5).